src/components/Survey/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+
+export const SurveyTypeEnum = {
```

Comment:
This feels like it should be two separate components to me. The links, text, and markup are different

Reply:
I'm not sure I agree, I guess I could see something like this:

components/
  Surveys/
    AnythingWrong/
    ImproveEasi/

but that feels convoluted for this and I know too many nested directories isn't recommended. I don't really like this either though:

components/
  AnythingWrongSurvey/
  ImproveEasiSurvey/

Reply:
You can also have named exports from `Surveys/index.tsx` too.

```
export const AnythingWrong = () => {
 {...}
}

export const ImproveEasi = () => {
 {...}
}

// Usage
import { AnythingWrong, ImproveEasi } from 'components/surveys';
```

Reply:
I like that a lot, thanks ill make the change